utils/catatonit/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=catatonit
+PKG_VERSION:=0.1.7
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/openSUSE/catatonit.git
+PKG_SOURCE_DATE:=2022-03-07
+PKG_SOURCE_VERSION:=d8d72fea155c144ed3bf298a35a1aba5625a5656
+PKG_MIRROR_HASH:=skip
```

Comment:
Why don't use tagged release?

Reply:
because in podman instructions (build script for catatonit in unpacked source), git tree is fetched instead of release; can't wonder if there is a reason they did it like that. Although, I didn't bother to check how close of latest commit is that release, nor how far their chosen commit is from current. Anyway, never touch a working system...

Reply:
It seems that there are not too many changes between 0.1.7 release and latest commit, but all those seem to be related to issues with autoconf files, so I suggest we go with them. Latest commit is dated in 2021, so most likely commit used by build script included with podman sources, is using the same version (without actually checking this..)

Reply:
From my point of view, using tagged releases is preferred and this commit https://github.com/openSUSE/catatonit/commit/99bb9048f532257f3a2c3856cfa19fe957ab6cec could be backported manually, but this is fine as it is.